baseframe/__init__.py

```diff
@@ -43,6 +43,15 @@
     'DOGPILE_CACHE_REGIONS': [('default', 3600)]
 }
 
+THEME_FILES = {
+    'bootstrap3': {'ajaxform.html.jinja2': 'baseframe/bootstrap3/ajaxform.html.jinja2', 'autoform.html.jinja2': 'baseframe/bootstrap3/autoform.html.jinja2',
+        'delete.html.jinja2': 'baseframe/bootstrap3/delete.html.jinja2', 'message.html.jinja2': 'baseframe/bootstrap3/message.html.jinja2',
+        'redirect.html.jinja2': 'baseframe/bootstrap3/redirect.html.jinja2'},
+    'mui': {'ajaxform.html.jinja2': 'baseframe/mui/ajaxform.html.jinja2', 'autoform.html.jinja2': 'baseframe/mui/autoform.html.jinja2',
+        'delete.html.jinja2': 'baseframe/mui/delete.html.jinja2', 'message.html.jinja2': 'baseframe/mui/message.html.jinja2',
+        'redirect.html.jinja2': 'baseframe/mui/redirect.html.jinja2'}
+}
```

Comment:
A note on indentation: one file per line, and the closing `}` should be indented (it violates PEP8 but is part of our style guide).

Reply:
Fixed